backend/schemas/sources/Base.libsonnet

```diff
@@ -18,6 +18,9 @@ local Types = {
   array: { type: 'array' },
   number: { type: 'number' },
   NULL: { type: Const.NULL },
+  boolean: { type: 'boolean' },
```

Comment:
We'll pick this up in a future consolidation of schemas (we have them running around the entire codebase)... I've got a "Compound" type for compound types. These could, perhaps, be called "atomic" types. However, this is not worth changing now, as we need to refactor all of the schemas and get them in one place in the tree. We'll pick that up in a future sprint *real soon now*.